sanic/cli/app.py

```diff
@@ -58,16 +56,27 @@ def __init__(self) -> None:
         self.main_process = (
             os.environ.get("SANIC_RELOADER_PROCESS", "") != "true"
         )
-        self.args: List[Any] = []
+        self.args: Namespace = Namespace()
         self.groups: List[Group] = []
+        self.inspecting = False
 
     def attach(self):
+        if sys.argv[1] == "inspect":
+            self.inspecting = True
+            self.parser.description = get_logo(True)
+            make_inspector_parser(self.parser)
+            return
+
```

Comment:
Can't we do this with subcommand groups instead?

Reply:
I thought about this and doing a similar hack like this to allow a static and dynamic value in the same location. But I do not like what it does to the help docs, and it makes the whole CLI app bleed a bit too much. In this location I think it works okay. When we are talking about the module, not so much. By keeping the parser attachments separate, it is cleaner to maintain.

![image](https://user-images.githubusercontent.com/166269/207968897-5c894b49-1131-400a-93c8-e605acb1f95b.png)